src/util/orders.ts

```diff
@@ -35,3 +38,33 @@ export const buildOrder = (params: BuildOrderParams, side: OrderSide): Order =>
         senderAddress: '0x0000000000000000000000000000000000000000',
     };
 };
+
+export const orderDetailsFeeEther = (
+    orderPrice: BigNumber,
+    orderType: OrderSide,
+    tokenDecimals: number = 18,
+): BigNumber => {
+    let fee = new BigNumber(1);
+    if (orderType === OrderSide.Buy) {
+        fee = new BigNumber(MAKER_FEE);
+    }
+
+    if (orderType === OrderSide.Sell) {
+        fee = new BigNumber(TAKER_FEE);
+    }
+
+    // Fee is a percentage
+    const orderPriceConverted = tokenAmountInUnits(orderPrice, tokenDecimals);
+    return fee.mul(orderPriceConverted).div(100);
```

Comment:
This doesn't seem correct. Here's how I understand the fees computation.

Both in buy and sell orders, you have a maker and a taker amount. You multiply them by the maker fee and the taker fee, respectively. This will give you an amount in ETH and the other in the selected token. Which is which depends on the type of the order: in sell orders, the resulting taker fee will be in ETH; in buy orders, the maker fee will be in ETH. So you multiply the other value (the one in units of the selected token) by the price, to convert it to ETH. Finally you sum both values: that's the total fee.

Does this make sense?